model/task_lifecycle_test.go

```diff
@@ -920,6 +920,22 @@ func TestGetBuildStatus(t *testing.T) {
 		{Status: evergreen.TaskFailed},
 	}
 	assert.Equal(t, evergreen.BuildFailed, getBuildStatus(buildTasks))
+
+	// blocked tasks don't prevent the build from completing
```

Comment:
Should use a full sentence per style guide

Reply:
fixed